website/content/terminal/guides/intros/stocks/index.md

```diff
@@ -1,225 +1,364 @@
-| --------- | ---------------------------------- | ------------- | ---------- | ------------------------- | -------------- |
-| MSF.BR    | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Belgium        |
-| MSF.DE    | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Germany        |
-| MSF.F     | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Germany        |
-| MSF2.L    | Leverage Shares 2x Microsoft ETC A | None          | None       | None                      | United-Kingdom |
-| MSF3.L    | Leverage Shares 3x Microsoft ETC   | None          | None       | None                      | United-Kingdom |
-| MSFS.L    | Leverage Shares -1x Microsoft ETC  | None          | None       | None                      | United-Kingdom |
-| MSFT.BA   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Argentina      |
-| MSFT.MX   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Mexico         |
-| MSFT      | Microsoft Corporation              | United States | Technology | Software - Infrastructure | USA            |
-| MSFT.SN   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Chile          |
-| MSFT.VI   | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Austria        |
-| MSFT34.SA | Microsoft Corporation              | United States | Technology | Software - Infrastructure | Brazil         |
-| SMSF.L    | Leverage Shares -1x Microsoft ETC  | None          | None       | None                      | United-Kingdom |
+### Codes
+
+`codes` prints the CIK, Composite FIGI, Share Class FIGI, and SIC codes - when available - for a US-listed stock. This function requires a free API key from Polygon.
+
+```console
+load aapl/codes
+
+            AAPL Codes             
+┏━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃                  ┃              ┃
+┡━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ CIK              │ 0000320193   │
+├──────────────────┼──────────────┤
+│ COMPOSITE FIGI   │ BBG000B9XRY4 │
+├──────────────────┼──────────────┤
+│ SHARE CLASS FIGI │ BBG001S5N8V8 │
+├──────────────────┼──────────────┤
+│ SIC CODE         │ 3571         │
+└──────────────────┴──────────────┘
+```
 
-Then, load in the historical data of the company by using the <a href="/terminal/reference/stocks/load/" target="_blank" rel="noreferrer noopener">load</a>. We also choose a different starting period by using `-s` as shown in the documentation:
+### Search
+
+The `search` function provides a way to find stocks by name, region, sector, industry and exchange location. The results can be easily exported as a CSV, JSON, or XLSX file.
+
+
+```console
+search -s technology --country india -e india --export csv
```

Comment:
Let's place this in a separate code block so people can copy.

Reply:
done